Playbooks/AS-PagerDuty-Integration/README.md

```diff
@@ -0,0 +1,136 @@
+# AS-PagerDuty-Integration
+
+Author: Accelerynt
+
+For any technical questions, please contact info@accelerynt.com  
+
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FAzure-Sentinel%2Fmaster%2FPlaybooks%2FAS-PagerDuty-Integration%2Fmain%2Fazuredeploy.json" target="_blank">
+    <img src="https://aka.ms/deploytoazurebutton""/>
+</a>
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2FAzure-Sentinel%2Fmaster%2FPlaybooks%2FAS-PagerDuty-Integration%2Fmain%2Fazuredeploy.json" target="_blank">
+<img src="https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/1-CONTRIBUTION-GUIDE/images/deploytoazuregov.png"/>
+</a>
+
+
+This playbook is intended to be run from a Microsoft Sentinel incident. It will create a PagerDuty event with the related Microsoft Sentinel incident and entity information.
+                                                                                                                                     
+![PagerDuty_Demo](Images/PagerDuty_Demo.png)
+ 
+                                                                                                                                     
+#
+### Requirements
+                                                                                                                                     
+The following items are required under the template settings during deployment: 
+
+* A PagerDuty integration key
+                                                                                                                                    
+# 
+### Setup
+                                                                                                                                     
+To Create a PagerDuty Integration Key:
+ 
+Log into your PagerDuty account and under the "**Services**" menu option, select "**Service Directory**". From the page, click the "**New Service**" button.
```

Comment:
Link to PagerDuty login page is missing.

Reply:
Resolved